### DISCUSSION TITLE
Option to hide Parent Object Name in Export

### DISCUSSION THREAD

**Discussion by jeff-restlesslabs** [tprouvot/Salesforce-Inspector-reloaded#335] [Ideas]
Sometimes It is very helpful to have the the Parent Object as a link in the Export fields when using related fields.
However when you are outputting a large number of fields or analyzing externally this can get in the way.
It would be great if you could include a setting to hide this in the Export

**Comment by tprouvot** [comment]
Hi @jeff-restlesslabs 
It has been delivered in https://github.com/tprouvot/Salesforce-Inspector-reloaded/issues/352 and is available in the beta version

**Reply by jeff-restlesslabs** [reply]
Awesome!  Is there a link where we can donate?

**Reply by tprouvot** [reply]
Thanks for this suggestion @jeff-restlesslabs ❤️ 
I was uncomfortable with the idea but I'm considering to provide a way to do it.

**Reply by jeff-restlesslabs** [reply]
You should get comfortable :). You have a great application that I know in my circle is widely used. 
Please put up a buy me a coffee link.
Also where can I access the beta releases?  I would be happy to help test

**Reply by tprouvot** [reply]
Thanks for those kind words 🙏 

Beta version is basically the next version where the new functionalities are tested during a month before releasing the main version
You can download it here 
https://chromewebstore.google.com/detail/salesforce-inspector-relo/lopjgjcglnncikiocpacfdbkmpbfmkcf
